### PR TITLE
some small changes

### PR DESCRIPTION
**Pull request by jklynch**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -46,18 +46,18 @@ Vagrant.configure("2") do |config|
     # create some directories for the sirepo docker container to use
     cd /home/vagrant
     mkdir -p .local/share/intake
-    chown -R vagrant:vagrant .local/share/intake
+    chown -R vagrant:vagrant .local
```
Comment (mrakitin) [inline Vagrantfile:49]:
```suggestion
    chown -Rv vagrant:vagrant .local
```

File: Vagrantfile [hunk 2]
```diff
@@ -46,18 +46,18 @@ Vagrant.configure("2") do |config|
...
     mkdir -p tmp/sirepo-docker-run
-    chown -R vagrant:vagrant tmp/sirepo-docker-run
+    chown -R vagrant:vagrant tmp
```
Comment (mrakitin) [inline Vagrantfile:51]:
```suggestion
    chown -Rv vagrant:vagrant tmp
```

**Review by mrakitin — COMMENTED** [review]
Should we get a similar update for the `sirepo-bluesky` library?

**Review by jennmald — APPROVED** [review]

**Comment by jklynch** [comment]
@mrakitin when this is working we can see if there is something here that should be in sirepo-bluesky

**Pull request merged**